### ISSUE TITLE
Question - How to most efficiently re-order sections?

### ISSUE THREAD

**Issue by aaronschubert0** [facebook/componentkit#196]
Currently I'm just deleting the old sections and then inserting new sections with new objects, like so:
```
             CKArrayControllerInputItems items;
             CKArrayControllerSections sections;
             for (int i = 0; i < completeSections.count; i++) {
                 NSArray *components = completeSections[i];
                 sections.remove(i);
                 sections.insert(i);
                 for (int j = 0; j < components.count; j++) {
                     id object = components[j];
                    items.insert([NSIndexPath indexPathForItem:j inSection:i], object);
                 }
             }
```

This seems quite in-efficent, is there a better way to do this?

**Comment by benlodotcom** [comment]
TL;DR not at the moment

Currently reorder/move of sections is not a supported operation and this would be the most efficient way to implement what you are doing. Reordering/moves for items and should be there shortly but we haven't look at section moves yet.
